7zip/README.md

```diff
@@ -1,15 +1,20 @@
 # 7zip
 
-7-Zip is a file archiver with a high compression ratio
+7-Zip is a file archiver with a high compression ratio.
 
 ## Maintainers
 
-* The Habitat Maintainers: <humans@habitat.sh>
+The Habitat Maintainers: <humans@habitat.sh>
 
 ## Type of Package
 
-Binary package
+Binary package.
```

Comment:
not a huge deal but not sure we need a period here since its not a sentence.  :)